docs/content/comparisons/druid-vs-storage-formats.md

```diff
@@ -0,0 +1,17 @@
+---
+layout: doc_page
+---
+
+Druid vs Storage Formats (Parquet/Kudu)
+=======================================
+
```

Comment:
feel free to ping me with questions about Kudu. Would be interesting to run some benchmarks and see how they go. I think your agg pushdown is a good advantage we don't have.